src/__tests__/fixtures/regression.css

```diff
@@ -0,0 +1,7 @@
+@custom-selector :--button .button;
+
+:--button {
+  &--active {
```

Comment:
From what I read in spec http://tabatkins.github.io/specs/css-nesting/#direct `&--active` selector is invalid. Not sure why postcss-nesting transform this. poke @jonathantneal